recipes/f77_zmq/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "f77_zmq" %}
+{% set version = "4.3.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/zeromq/f77_zmq/releases/download/v4.3.2_autotools/f77-zmq-4.3.2.tar.gz
+  sha256: e027bf1017ca6b47ce4a3481ae5b255db8ea86294bc32f1207bd90dbdece7671
+
+build:
+  skip: true  # [win]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('fortran') }}
+    - make
+  host:
+    - zeromq
+  run:
+    - zeromq
+
+test:
+  commands:
+    - test -f ${PREFIX}/lib/libf77zmq.a   # [unix]
```

Comment:
According to cfep 13 we should avoid static libraries. Is there a dylib on macOS?

Reply:
OK. Yes, there is a dylib on MacOS. I replaced that line with:
```
    - test -f ${PREFIX}/lib/libf77zmq.dylib  # [osx]                                                                          
```

Reply:
@wolv ping!